boardwalk/conf/boardwalk.config.template

```diff
@@ -1,3 +1,6 @@
+azul_s3_bucket={{ AZUL_S3_BUCKET }}
+user_aws_key_id={{ AWS_ACCESS_KEY_ID }}
+user_aws_secret_key={{ AWS_SECRET_ACCESS_KEY }}
```

Comment:
I think you should match the names how it already is in the develop branch. That way when we eventually reconcile the two branches, it will be less confusing.

https://github.com/DataBiosphere/cgp-deployment/blob/develop/boardwalk/conf/boardwalk.config.template#L22

Reply:
I actually followed the naming in `develop`, but I started with `install_bootstrap`.

https://github.com/DataBiosphere/cgp-deployment/blob/develop/install_bootstrap#L230
https://github.com/DataBiosphere/cgp-deployment/blob/develop/install_bootstrap#L397

The problem is that in `develop` the same variable is named differently in different files. In fact, some of the variable names change within `install_bootstrap`, depending for what service the user is queried for. For example, compare these two lines:

https://github.com/DataBiosphere/cgp-deployment/blob/develop/install_bootstrap#L397
https://github.com/DataBiosphere/cgp-deployment/blob/develop/install_bootstrap#L942

I'm not sure if this is by design (in which case I'm missing the intention / utility)? I can use the names from the `boardwalk.config.template`. I had to make a decision whether to consistently name them in as in `install_bootstrap` or as in the mustache template.

Reply:
You're right. I think the variable name is intentionally different than the environment variable name on purpose. The way you have it is good.

Sorry that I sent you down the wrong path on this.

I think you should just copy the code as it is in develop branch.

Reply:
* I made the names for the AWS access key ID and secret access key consistent across all files.
* Amended the README.